config/locales/en.yml

```diff
@@ -1624,7 +1624,7 @@ two_factor_authentication.backup_codes.warning_html: '<strong>You’ve only set
 two_factor_authentication.choose_another_option: '‹ Choose another authentication method'
 two_factor_authentication.confirm_backup_code_setup_content_html:
   - '<strong>Backup codes are the least preferred authentication method</strong> because the codes can easily be lost. Try a safer option, like an authentication application or a security key.'
-  - We’ll give you 10 codes that you can download, print, copy or write down. You’ll enter one code every time you sign in.
+  - We’ll give you %{number_of_codes} codes that you can download, print, copy or write down. You’ll enter one code every time you sign in.
```

Comment:
I'm guessing we will always have more than 1 backup code, but this is something we could consider using `one:/other:` count syntax for? (no idea how that applies within an array, that sounds like a messy issue for another time)

Reply:
Hm, in retrospect yes. But also, I don't think we have translations to be able to do that at this point, and I'd be reluctant to go through that process again now. 😕 

Separately, I also want to update [the remaining hard-coded number value](https://github.com/18F/identity-idp/pull/10970/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R817). That's a little trickier as well, since its spelled out, and also spelled out inconsistently across the languages. (e.g. spelled-out "five" in English, but numeric "10" in Chinese).

Reply:
Maybe we could use a gem like https://github.com/kslazarev/numbers_and_words to get the written out names? I know a common rule of thumb in English is to write out numbers less than 10, but I'm not sure what our style guide is for other locales

Reply:
> Maybe we could use a gem like https://github.com/kslazarev/numbers_and_words to get the written out names? I know a common rule of thumb in English is to write out numbers less than 10, but I'm not sure what our style guide is for other locales

I pushed a commit shortly before your comment, but yeah, was also struggling to find commonalities in other languages, especially since we have come inconsistencies within Spanish for example. What I landed at is "spelled out" in English, French, and Spanish, while using the numeric value in Chinese.

Reply:
Unless we see number translation complexity going up, I'm kind of hesitant to bring in a gem to do it.